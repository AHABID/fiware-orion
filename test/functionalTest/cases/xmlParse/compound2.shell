#   2. ok:  One string 's1' == 'STRING' as compound value (contextValue is a struct)
echo "2: +++++++++++++++++++++++++++++++++"
url='/ngsi10/updateContext'
payload='<?xml version="1.0"?>
<updateContextRequest>
  <contextElementList>
    <contextElement>
      <entityId type="Room" isPattern="false">
        <id>ConferenceRoom</id>
      </entityId>
      <contextAttributeList>
        <contextAttribute>
          <name>temperature</name>
          <type>degree</type>
          <contextValue>
            <s1>STRING</s1>
          </contextValue>
        </contextAttribute>
      </contextAttributeList>
    </contextElement>
  </contextElementList>
  <updateAction>APPEND</updateAction>
</updateContextRequest>'
curlXml ${url} "${payload}"

