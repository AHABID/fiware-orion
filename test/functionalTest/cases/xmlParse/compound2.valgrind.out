==26814== Memcheck, a memory error detector
==26814== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==26814== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==26814== Command: contextBroker -vvvvv -harakiri -port 9999 -pidpath /tmp/orion_9999.pid -db testharness -t 0-1
==26814== 
T@09:24:58  paConfigActions[727]: setting trace levels to '0-1'
E@09:24:58  exitFunc[559]: unlink(/tmp/orion_9999.pid): No such file or directory
==26814== 
==26814== HEAP SUMMARY:
==26814==     in use at exit: 17,690 bytes in 104 blocks
==26814==   total heap usage: 3,267 allocs, 3,163 frees, 281,093 bytes allocated
==26814== 
==26814== LEAK SUMMARY:
==26814==    definitely lost: 0 bytes in 0 blocks
==26814==    indirectly lost: 0 bytes in 0 blocks
==26814==      possibly lost: 0 bytes in 0 blocks
==26814==    still reachable: 17,690 bytes in 104 blocks
==26814==         suppressed: 0 bytes in 0 blocks
==26814== Rerun with --leak-check=full to see details of leaked memory
==26814== 
==26814== For counts of detected and suppressed errors, rerun with: -v
==26814== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 7 from 7)
F@09:24:58  ***[600]: Connected to mongo at localhost:testharness
==26819== Source and destination overlap in strncpy(0xa4d8a7, 0xa4d8a7, 63)
==26819==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==26819==    by 0x659678: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:600)
==26819==    by 0x5DF38D: main (contextBroker.cpp:737)
==26819== 
==26819== Source and destination overlap in strncpy(0xa4d8e2, 0xa4d8e2, 63)
==26819==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==26819==    by 0x65969C: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:603)
==26819==    by 0x5DF38D: main (contextBroker.cpp:737)
==26819== 
V@09:24:58  restStart[499]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@09:24:58  restStart[541]: Starting HTTP daemon on IPv6 :: port 9999
==26819== Thread 4:
==26819== Syscall param mmap(length) contains uninitialised byte(s)
==26819==    at 0x58B9AFA: mmap (syscall-template.S:82)
==26819==    by 0x4E398E7: MHD_pool_create (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35F47: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E33C62: MHD_set_connection_value (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34033: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E3426D: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34783: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
2@09:25:06  xmlRequestGet[155]: Found xmlRequest of type 17, method 'POST' - index 12 (/updateContextRequest)
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x63FD37: contextAttributeCustomMetadataToBson(mongo::BSONObj&, ContextAttribute*) (MongoCommonUpdate.cpp:536)
==26819==    by 0x649675: createEntity(EntityId, ContextAttributeVector, std::string*) (MongoCommonUpdate.cpp:1254)
==26819==    by 0x64D4B3: processContextElement(ContextElement*, UpdateContextResponse*, std::string) (MongoCommonUpdate.cpp:1594)
==26819==    by 0x633A5B: mongoUpdateContext(UpdateContextRequest*, UpdateContextResponse*) (mongoUpdateContext.cpp:51)
==26819==    by 0x615FBC: postUpdateContext(ConnectionInfo*, int, std::vector<std::string, std::allocator<std::string> >, ParseData*) (postUpdateContext.cpp:44)
==26819==    by 0x65D126: restService(ConnectionInfo*, RestService*) (RestService.cpp:155)
==26819==    by 0x6577FC: serve(ConnectionInfo*) (rest.cpp:234)
==26819==    by 0x658AA1: connectionTreat(void*, MHD_Connection*, char const*, char const*, char const*, char const*, unsigned long*, void**) (rest.cpp:458)
==26819==    by 0x4E34328: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DBF: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E335B6: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E335BD: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2A639: memcpy (mc_replace_strmem.c:121)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2A64E: memcpy (mc_replace_strmem.c:838)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2A69B: memcpy (mc_replace_strmem.c:838)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2A790: memcpy (mc_replace_strmem.c:838)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Use of uninitialised value of size 8
==26819==    at 0x4C2A7C8: memcpy (mc_replace_strmem.c:838)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Use of uninitialised value of size 8
==26819==    at 0x4C2A7ED: memcpy (mc_replace_strmem.c:838)
==26819==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Use of uninitialised value of size 8
==26819==    at 0x58CFC9C: __vsprintf_chk (vsprintf_chk.c:79)
==26819==    by 0x58CFC0E: __sprintf_chk (sprintf_chk.c:33)
==26819==    by 0x4E33622: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Use of uninitialised value of size 8
==26819==    at 0x4C2953C: strcpy (mc_replace_strmem.c:429)
==26819==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C29569: strcpy (mc_replace_strmem.c:112)
==26819==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2956E: strcpy (mc_replace_strmem.c:121)
==26819==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4C2957F: strcpy (mc_replace_strmem.c:429)
==26819==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Use of uninitialised value of size 8
==26819==    at 0x4E3363E: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Syscall param sendto(msg) contains uninitialised byte(s)
==26819==    at 0x58BE0AC: send (send.c:32)
==26819==    by 0x4E36D69: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35963: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35A87: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A40: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Conditional jump or move depends on uninitialised value(s)
==26819==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
==26819== Thread 3:
==26819== Syscall param munmap(length) contains uninitialised byte(s)
==26819==    at 0x58B9B27: munmap (syscall-template.S:82)
==26819==    by 0x4E3996B: MHD_pool_destroy (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E37030: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x4E3881C: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==26819==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==26819==    by 0x58BD0EC: clone (clone.S:112)
==26819== 
E@09:25:06  orionExit[548]: Received a 'DIE' request
==26819== 
==26819== HEAP SUMMARY:
==26819==     in use at exit: 23,191 bytes in 148 blocks
==26819==   total heap usage: 3,838 allocs, 3,690 frees, 484,221 bytes allocated
==26819== 
==26819== LEAK SUMMARY:
==26819==    definitely lost: 0 bytes in 0 blocks
==26819==    indirectly lost: 0 bytes in 0 blocks
==26819==      possibly lost: 1,394 bytes in 16 blocks
==26819==    still reachable: 21,797 bytes in 132 blocks
==26819==         suppressed: 0 bytes in 0 blocks
==26819== Rerun with --leak-check=full to see details of leaked memory
==26819== 
==26819== For counts of detected and suppressed errors, rerun with: -v
==26819== Use --track-origins=yes to see where uninitialised values come from
==26819== ERROR SUMMARY: 43 errors from 30 contexts (suppressed: 7 from 7)
